modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpClientDiscoverySpiSelfTest.java

```diff
@@ -2578,15 +2578,23 @@ private void pauseResumeOperation(boolean isPause, AtomicBoolean... locks) {
                 afterWrite.apply(msg, sock);
         }
 
-        /** {@inheritDoc} */
```

Comment:
Why replaced the inherited doc?

Reply:
fixed. i've reviewd code obe more time and i've veririden method instead of overloading. so doc is inherited now.
TcpDiscoveryConnectionCheckMessage is used as a stub instead of msg which was pased via parameter according to your comment:
// Replace logic routine message with a stub to update last-sent-time to avoid segmentation on
// connRecoveryTimeout.

Reply:
I'm afraid, sending TcpDiscoveryConnectionCheckMessage won't help us here. Such trick was involved by me accurately where it could work. Unfortunately, it not going to work everywhere. DiscoverySpi is quite complex.

Reply:
so what's better to use  In this particular case? null?

Reply:
I see particular case as understanding what failed test does and how to fix it. Or, at least, how to locally return broken logic. Skipping certain messages in this case, not skipping others. I would start with logging messages in failed test on master-branch. Which of them are skipped and what is the corresponding raw data (byte[]). We might slightly re-write skipping messages basing on the raw data, not on the message unused.